example/k8s-aliyun-ubuntu/k8s-master.yaml

```diff
@@ -0,0 +1,233 @@
+apiVersion: archon.kubeup.com/v1
+kind: InstanceGroup
+metadata:
+  name: k8s-master
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: k8s-master
+  template:
+    metadata:
+      labels:
+        app: k8s-master
+      annotations:
+        initializers: archon.kubeup.com/private-ip,archon.kubeup.com/public-ip
+        aliyun.archon.kubeup.com/use-ssh: "true"
+    spec:
+      networkName: k8s-net
+      instanceType: ecs.n1.tiny
+      os: Ubuntu
+      image: ubuntu_16_0402_64_40G_base_20170222.vhd
+      files:
+      - name: apt-pgp 
+        path: "/tmp/kubernetes.pgp"
+        content: |
+          -----BEGIN PGP PUBLIC KEY BLOCK-----
+          
+          mQENBFUd6rIBCAD6mhKRHDn3UrCeLDp7U5IE7AhhrOCPpqGF7mfTemZYHf/5Jdjx
+          cOxoSFlK7zwmFr3lVqJ+tJ9L1wd1K6P7RrtaNwCiZyeNPf/Y86AJ5NJwBe0VD0xH
+          TXzPNTqRSByVYtdN94NoltXUYFAAPZYQls0x0nUD1hLMlOlC2HdTPrD1PMCnYq/N
+          uL/Vk8sWrcUt4DIS+0RDQ8tKKe5PSV0+PnmaJvdF5CKawhh0qGTklS2MXTyKFoqj
+          XgYDfY2EodI9ogT/LGr9Lm/+u4OFPvmN9VN6UG+s0DgJjWvpbmuHL/ZIRwMEn/tp
+          uneaLTO7h1dCrXC849PiJ8wSkGzBnuJQUbXnABEBAAG0QEdvb2dsZSBDbG91ZCBQ
+          YWNrYWdlcyBBdXRvbWF0aWMgU2lnbmluZyBLZXkgPGdjLXRlYW1AZ29vZ2xlLmNv
+          bT6JAT4EEwECACgFAlUd6rICGy8FCQWjmoAGCwkIBwMCBhUIAgkKCwQWAgMBAh4B
+          AheAAAoJEDdGwginMXsPcLcIAKi2yNhJMbu4zWQ2tM/rJFovazcY28MF2rDWGOnc
+          9giHXOH0/BoMBcd8rw0lgjmOosBdM2JT0HWZIxC/Gdt7NSRA0WOlJe04u82/o3OH
+          WDgTdm9MS42noSP0mvNzNALBbQnlZHU0kvt3sV1YsnrxljoIuvxKWLLwren/GVsh
+          FLPwONjw3f9Fan6GWxJyn/dkX3OSUGaduzcygw51vksBQiUZLCD2Tlxyr9NvkZYT
+          qiaWW78L6regvATsLc9L/dQUiSMQZIK6NglmHE+cuSaoK0H4ruNKeTiQUw/EGFaL
+          ecay6Qy/s3Hk7K0QLd+gl0hZ1w1VzIeXLo2BRlqnjOYFX4A=
+          =HVTm
+          -----END PGP PUBLIC KEY BLOCK-----
+      - name: run-kubeadm
+        path: "/tmp/run-kubeadm"
+        permissions: "0755"
+        template: |
+          cloud-init-per once aptaddsource apt-key add /tmp/kubernetes.pgp
```

Comment:
why don't we use the add repo and install package feature included in cloud-init